src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -56,6 +56,8 @@
 
 EVENT_HANDLER = Callable[[List[CloudEvent]], Awaitable[None]]
 
+CLOSE_EVENT_SENTINEL = object()
```

Comment:
Should we make this into a dummy class, so that we can type the queue more properly than  `Any`